neuron/neuron.cabal

```diff
@@ -48,8 +48,8 @@ common library-common
     filepattern,
     megaparsec,
     commonmark ^>= 0.1.0.0,
```

Comment:
Shouldn't `commonmark` have a version constraint similar to `commonmark-extensions` and `commonmark-pandoc`?

Reply:
Apparently jgm is not keeping the version numbers in sync.